sdk/python/tests/test_kfp_samples.sh

```diff
@@ -117,10 +122,10 @@ if [ ! -d "${VENV_DIR}" ]; then
 fi
 source "${VENV_DIR}/bin/activate"
 
-# install KFP with the desired KFP_VERSION (unless already installed)
-if ! (pip show "kfp" | grep Version | grep -q "${KFP_VERSION}"); then
-  echo "Installing KFP ${KFP_VERSION} ..."
-  pip install -q -e "${KFP_CLONE_DIR}/sdk/python"
+# install KFP with the desired KFP SDK version (unless already installed)
+if ! (pip show "kfp" | grep Version | grep -q "${KFP_SDK_VERSION}"); then
+  echo "Installing KFP SDK ${KFP_SDK_VERSION} ..."
+  pip install -q kfp==${KFP_SDK_VERSION}
```

Comment:
So just want to double check here. Previous it's `pip install -q -e "${KFP_CLONE_DIR}/sdk/python"`. Is this the expected changes?

Reply:
yeah, testing the install from pip here (`kfp==1.6.3`), otherwise we get the KFP `1.5.0` SDK from the KFP git clone

Reply:
We want the KFP testdata pipelines from the KFP clone with `KFP_GIT_VERSION` == `1.5.0` but the kfp SDK from PyPI with `KFP_SDK_VERSION` == `1.6.3`